proto/internalapi/scanner/v4/vulnerability_report.proto

```diff
@@ -33,23 +33,29 @@ message VulnerabilityReport {
       V3 v3 = 2;
     }
     string id = 1;
-    string name = 3;
-    string description = 4;
-    google.protobuf.Timestamp issued = 5;
-    string link = 6;
-    string severity = 7;
-    Severity normalized_severity = 8;
-    string package_id = 9;
-    string distribution_id = 10;
-    string repository_id = 11;
-    string fixed_in_version = 12;
-    CVSS cvss = 13;
+    string name = 2;
```

Comment:
for some reason we skipped 2. Sine we haven't released this yet, I updated the tags